tests/test_app/urls.py

```diff
@@ -2,12 +2,17 @@
 from django.conf.urls.static import static
 from django.contrib import admin
 from django.conf.urls import url, include
-
+import django
+from distutils.version import StrictVersion
 admin.autodiscover()
 
+extra_url_kwargs = {'prefix': 'admin'} \
+    if StrictVersion(django.get_version()) < StrictVersion('1.7')\
+    else\
+    {}
```

Comment:
By the way, I would rename the variable `extra_url_kwargs` to `login_url_kwargs`.